spec/github_pages_health_check_domain_spec.rb

```diff
@@ -345,8 +353,8 @@ def cname_packet(domain)
       stub_request(:head, "http://developers.facebook.com").to_return(:status => 200, :headers => {})
       check = make_domain_check "developers.facebook.com"
       expect(check.valid?).to eql(false)
-      expect(check.reason.class).to eql(GitHubPages::HealthCheck::Errors::InvalidCNAMEError)
-      expect(check.reason.message).to match(/not set up with a correct CNAME record/i)
+      expect(check.reason.class).to eql(GitHubPages::HealthCheck::Errors::NotServedByPagesError)
+      expect(check.reason.message).to eql("Domain does not resolve to the GitHub Pages server")
```

Comment:
@benbalter Do you remember why this was an InvalidCNAMEError? Its records as of today include an MX (I think?) so it is an A record'd domain:

```text
developers.facebook.com.   3550   IN   CNAME   developers.c10r.facebook.com.
developers.c10r.facebook.com.   37   IN   A   31.13.77.6
developers.facebook.com.   3511   IN   CNAME   developers.c10r.facebook.com.
developers.c10r.facebook.com.   300   IN   MX   10 msgin.vvv.facebook.com.
```

Maybe we could use `code.facebook.com` or something else instead?